internal/access/identity.go

```diff
@@ -111,10 +111,10 @@ func ListIdentities(c *gin.Context, name string, groupID uid.ID, ids []uid.ID) (
 	}
 
 	if groupID != 0 {
-		return data.ListIdentitiesByGroup(db, groupID, selectors...)
+		return data.ListIdentitiesByGroup(db.Preload("Providers"), groupID, selectors...)
 	}
 
-	return data.ListIdentities(db, selectors...)
+	return data.ListIdentities(db.Preload("Providers"), selectors...)
```

Comment:
I really don't like that we don't have a way for the handler to request this preload. Now all users of ListIdentities get it whether they want it or not.